libraries/AP_SerialManager/AP_SerialManager.cpp

```diff
@@ -232,6 +232,7 @@ const AP_Param::GroupInfo AP_SerialManager::var_info[] = {
     // @DisplayName: Telem1 options
     // @Description: Control over UART options. The InvertRX option controls invert of the receive pin. The InvertTX option controls invert of the transmit pin. The HalfDuplex option controls half-duplex (onewire) mode, where both transmit and receive is done on the transmit wire. The Swap option allows the RX and TX pins to be swapped on STM32F7 based boards.
     // @Bitmask: 0:InvertRX, 1:InvertTX, 2:HalfDuplex, 3:Swap, 4: RX_PullDown, 5: RX_PullUp, 6: TX_PullDown, 7: TX_PullUp, 8: RX_NoDMA, 9: TX_NoDMA, 10: Don't forward mavlink to/from
+    // @Values: 0: None, 1:InvertRX, 2:InvertTX, 4:HalfDuplex, 8:Swap, 16: RX_PullDown, 32: RX_PullUp, 64: TX_PullDown, 128: TX_PullUp, 256: RX_NoDMA, 512: TX_NoDMA, 1024: Don't forward mavlink to/from
```

Comment:
I'd remove the space between "0:" and "None" but not a big deal

Reply:
fixed